core/src/main/codegen/templates/Parser.jj

```diff
@@ -4313,6 +4316,11 @@ SqlNode StringLiteral() :
     // or comments may not occur between the prefix and the first quote, the
     // head fragment, with any prefix, is one token.
 
+<#if parser.allowHexCharacterLiteral>
+    hexCharLiteral = SqlHexCharStringLiteral()
+    {return hexCharLiteral;}
```

Comment:
nit: Spacing before/after open and close braces.

Reply:
Added in https://github.com/googleinterns/calcite/pull/155/commits/0d4b39999eaa58e5cdf353dad88355c79e0ab304